cmd/lint/main.go

```diff
@@ -19,10 +19,8 @@ var commandRules = []linter.CommandRule{
 	linter.RequireLowerCase("Use"),
 	linter.RequireRealWords("Use", '-'),
 	linter.Filter(linter.RequireSingular("Name"),
-		linter.ExcludeCommandContains("local services"),
 		linter.ExcludeCommand("kafka client-config create nodejs")),
 
-	linter.Filter(linter.RequireCapitalizeProperNouns("Short", properNouns), linter.ExcludeCommand("local current")),
```

Comment:
Can't suggest an edit here, but you're deleting the entire rule. It should be rewritten to the following instead of deleted:
```
linter.RequireCapitalizeProperNouns("Short", properNouns),
```